### ISSUE TITLE
Undo/Redo system

### ISSUE THREAD

**Issue by adriengivry** [adriengivry/Overload#41]
We should be able to undo and redo editor actions to make Overload less destructive. We can for example store editor commands in classes with a "Do" and "Undo" method to be able to undo any command. These commands can be saved into a stack after they are executed.

**Comment by adriengivry** [comment]
This feature is currently blocking the release of the 1.4 release, let's push back this one till the next milestone.

**Comment by SolarcGit** [comment]
I disagree undo redo would make making easier by leagues

**Comment by litelawliet** [comment]
Yes, but it's a big endeavour. I'm okay for it being pushed back to let that milestone pass and work on it for the next milestone.
If you'd like to work on that you're free to go :)
